diff --git a/language/it/acp/board.php b/language/it/acp/board.php
index b3c2fd6..0e6d26e 100644
--- a/language/it/acp/board.php
+++ b/language/it/acp/board.php
@@ -466,8 +466,8 @@ $lang = array_merge($lang, array(
 	'IP_LOGIN_LIMIT_MAX_EXPLAIN'	=> 'Dopo questo numero di tentativi di connessione falliti l’utente dovrà ulteriormente confermare il proprio accesso con l’aggiunta di una contromisura spambot. Inserendo 0, si disabilita questa funzione.',
 	'IP_LOGIN_LIMIT_TIME'			=> 'Limite di tempo per i tentativi di login per indirizzo IP',
 	'IP_LOGIN_LIMIT_TIME_EXPLAIN'	=> 'Limite di tempo per i tentativi di login che scadono dopo questo periodo.',
-	'IP_LOGIN_LIMIT_USE_FORWARDED_EXPLAIN'	=> ' Invece di limitare i tentativi di login tramite l’indirizzo IP, questi possono essere limitati dai valori di <var>X_FORWARDED_FOR</var>. <br /><em><strong>Attenzione:</strong> abilita questo modulo solo se si opera su un server proxy che imposta <var>X_FORWARDED_FOR</var> su valori affidabili.</em>',
-	'IP_LOGIN_LIMIT_USE_FORWARDED_EXPLAIN'	=> 'Invece di limitare i tentativi di login tramite indirizzo IP, questi possono essere limitati dai valori di <var>X_FORWARDED_FOR</var>.<br /><em><strong>Attenzione:</strong> abilita questa funzione solo se si utilizza un server proxy che imposta <var>X_FORWARDED_FOR</var> su valori affidabili.</em>',
+	'IP_LOGIN_LIMIT_USE_FORWARDED'	=> 'Limite di tentativi di login tramite l’intestazione <var>X_FORWARDED_FOR</var>',
+	'IP_LOGIN_LIMIT_USE_FORWARDED_EXPLAIN'	=> 'Invece di limitare i tentativi di login tramite indirizzo IP, questi possono essere limitati dai valori di <var>X_FORWARDED_FOR</var>.<br /><em><strong>Attenzione:</strong> abilita questa funzione solo se utilizzi un server proxy che imposta <var>X_FORWARDED_FOR</var> su valori affidabili.</em>',
 	'MAX_LOGIN_ATTEMPTS'			=> 'Numero massimo di tentativi di login',
 	'MAX_LOGIN_ATTEMPTS_EXPLAIN'	=> 'Il numero di tentativi di login consentiti per un singolo account prima che le contromisure spambot siano attivate. Inserisci 0 per evitare che le contromisure spambot siano attivate per gli account utente.',
 	'NO_IP_VALIDATION'				=> 'Nessuno',
